newreps/index.html

```diff
@@ -15,7 +15,14 @@
   <title>New Mozilla Reps</title>
 </head>
 <body>
-
+  
+<div class="banner">
+<div class="banner-content">
+<p>This page is not maintained anymore, see for the latest information about the Reps program; 
+<a href="https://discourse.mozilla.org/c/reps/9" > Discource</a></p>
```

Comment:
I'd suggest something like the following:

```
<p>This page is no longer maintained, check <a href="https://discourse.mozilla.org/c/reps/9">Discourse</a> for the latest information about the Reps program</p>
```

Reply:
Bonus points if indentation is followed ;)

Reply:
> I'd suggest something like the following:
> 
> ```
> <p>This page is no longer maintained, check <a href="https://discourse.mozilla.org/c/reps/9">Discourse</a> for the latest information about the Reps program</p>
> ```

yes better


> Bonus points if indentation is followed ;)

i will go for the points